cmd/intervalOverlap/intervalOverlap.go

```diff
@@ -58,7 +58,7 @@ func main() {
 	//var extend *int = flag.Int("extend", 0, "WIP") //TODO
 	var nonOverlap *bool = flag.Bool("nonOverlap", false, "Return records that do NOT have any overlap with records in the select file")
 	var threads *int = flag.Int("threads", 1, "Number of threads to use.")
-	//var percentOverlap *float64 = flag.Float64("percentOverlap", 0, "WIP") //TODO
+	var thresholdOverlap *float64 = flag.Float64("thresholdOverlap", 0, "Returns regions that have a minimum percentage of overlap between select and infile regions.")
```

Comment:
Would it help users to have a check that thresholdOverlap is between 0 and 1, and specify in the usage that we need a fraction (as opposed to percentage)?

Reply:
this is a one to one comparison not a total overlap, That was purposeful. Is there something in the language that you think is unclear or could be specified?